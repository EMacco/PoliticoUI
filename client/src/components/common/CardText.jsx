import React from 'react';
import PropTypes from 'prop-types';

const CardText = ({ title, value }) => {
  return (
    <label>
      {title}: <span className="profile-answers">{value}</span>
    </label>
  );
};

CardText.propTypes = {
  title: PropTypes.string.isRequired,
  value: PropTypes.string.isRequired
};

export default CardText;
