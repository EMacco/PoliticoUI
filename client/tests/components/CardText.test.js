import React from 'react';
import CardText from '@components/common/CardText';

const props = {
  title: 'Date',
  value: '12th July 2019'
};

const setup = () => {
  const wrapper = shallow(<CardText {...props} />);
  return wrapper;
};

describe('<CardText />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = setup();
  });

  it('should render the CardText component correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
    expect(wrapper.find('CardText')).toBeTruthy();
  });
});
