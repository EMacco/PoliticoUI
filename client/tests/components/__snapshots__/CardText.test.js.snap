// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardText /> should render the CardText component correctly 1`] = `
<label>
  Date
  : 
  <span
    className="profile-answers"
  >
    12th July 2019
  </span>
</label>
`;
